packages/admin/src/pages/scheme/[schemeId]/view-editors/index.page.tsx

```diff
@@ -0,0 +1,57 @@
+import InferProps from '../../../../types/InferProps';
+import { SummaryList } from 'gap-web-ui';
+import Meta from '../../../../components/layout/Meta';
+import CustomLink from '../../../../components/custom-link/CustomLink';
+import {
+  UnformattedEditorRow,
+  getServerSideProps,
+} from '../editors.getServerSideProps';
+import { Row } from 'gap-web-ui/dist/cjs/components/summary-list/SummaryList';
+
+export { getServerSideProps };
+
+const rmAction = (acc: Row[], row: UnformattedEditorRow) => [
+  ...acc,
+  { ...row, action: undefined },
+];
+
+const ViewEditors = ({
+  editorRows,
+  schemeName,
+  schemeId,
+}: InferProps<typeof getServerSideProps>) => {
+  const mappedTableRows = editorRows.reduce(rmAction, []);
```

Comment:
```suggestion

export { getServerSideProps };

const rmAction = (row: UnformattedEditorRow) => ({
  ...row,
  action: undefined,
});

const ViewEditors = ({
  editorRows,
  schemeName,
  schemeId,
}: InferProps<typeof getServerSideProps>) => {
  const mappedTableRows = editorRows.map(rmAction);
```

Reply:
Reduce is only needed over a map if you need to continuously use the previous map iterations response, here we only need the current element